projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt

```diff
@@ -522,12 +525,12 @@ class ProjectorServer private constructor(
       return
     }
 
-    val ipString = conn.remoteSocketAddress?.address?.hostAddress
+    val remoteAddress = conn.remoteSocketAddress?.address
+    check(remoteAddress != null) { "Remote address is null!" } // drop nullability
```

Comment:
I think we should handle this situation better because now a user can't do anything if they receive this "Remote address is null!" exception. And it seems possible that `address` contains `null` – we have a similar report: https://youtrack.jetbrains.com/issue/PRJ-212 . With this change, PRJ-212 can fail here too.

So my proposal is to handle nullability: if it's null, we can output something like `Somebody (can't determine the host's name or IP) wants to connect with $accessType access. Allow the connection?`